roslibrust/examples/native_ros1.rs

```diff
@@ -4,7 +4,7 @@
 
 #[cfg(feature = "ros1")]
 #[tokio::main]
-async fn main() -> Result<(), Box<dyn std::error::Error>> {
+async fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync>> {
```

Comment:
Why make the return of `main` have these bounds? I think it's just noise.

Reply:
I think we should punt to #118 

I ended up having to add this bound in a number of places to make it valid to return the errors from tasks.